### PR TITLE
chore: ignore incall hand raise - WPB-14922

### PR DESCRIPTION
**Pull request by netbe**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-14922" title="WPB-14922" target="_blank">TaskWPB-14922</a>  Ignore Hande raising protobuf message from Web
  </td></table>
  <br />
 

### Issue

* update protobufs
* Ignores incall hand raise

### Testing

* [x] test with call in the web

### CODE REVIEW

File: wire-ios-protos/Protos/messages.pb.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 //
 // Wire
-// Copyright (C) 2024 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (caldrian) [inline wire-ios-protos/Protos/messages.pb.swift]: I guess we need to update the header somewhere else, right?

File: wire-ios-protos/Protos/messages.pb.swift [hunk 2]
```diff
@@ -2318,6 +2331,28 @@ public struct InCallEmoji: Sendable {
...
 
+public struct InCallHandRaise: Sendable {
+  // SwiftProtobuf.Message conformance is added in an extension below. See the
+  // `Message` and `Message+*Additions` files in the SwiftProtobuf library for
+  // methods supported on all messages.
+
+  /// true if the hand is raised, false if lowered
+  public var isHandUp: Bool {
+    get {return _isHandUp ?? false}
+    set {_isHandUp = newValue}
```
Comment (caldrian) [inline wire-ios-protos/Protos/messages.pb.swift:2325]: Will/should this be formatted properly?

**Review by caldrian — APPROVED** [review]
Looks fine for me, however, it's the first time I'm getting in touch with this code.

File: wire-ios-protos/Protos/messages.pb.swift (hunk 2 above)
Comment (johnxnguyen) [reply wire-ios-protos/Protos/messages.pb.swift:2325]: It's generated by the swift protobuf tool.

**Review by johnxnguyen — APPROVED** [review]

File: wire-ios-protos/Protos/messages.pb.swift (hunk 2 above)
Comment (netbe) [reply wire-ios-protos/Protos/messages.pb.swift:2325]: this is generated code by protoc, so I wouldn't format it, I am surprise SwiftFormat did not touch the files though

File: wire-ios-protos/Protos/messages.pb.swift (hunk 1 above)
Comment (netbe) [reply wire-ios-protos/Protos/messages.pb.swift]: https://github.com/wireapp/generic-message-proto

**Comment by samwyndham** [comment]
Removing myself as reviewer as this PR is approved

**Pull request merged**